bin/pre-commit.sh

```diff
@@ -19,7 +19,12 @@ go vet -all ./...
 rc=$((rc || $?))
 
 echo "Running go test"
-go test -v ./...
+go list ./... | grep -vF pkg/framework/test | xargs go test -v
```

Comment:
I think there should be some sort of "default" value for `KUBE_ASSERT_DIR` and maybe we can download the binaries before running `go test`?

Reply:
Agreed. I think this also ties back to making our think work well with `go test`. As soon as we have that we can retire our test wrapper script `pkg/framework/test/scripts/run-tests.sh` and think about better ways of handling the test binaries/assets resp. their location. We in fact have all those things in our backlog.